package.json

```diff
@@ -46,13 +43,17 @@
     "@eslint/js": "~9.23.0",
     "@reporters/github": "^1.7.2",
     "@testing-library/react": "~16.3.0",
-    "@types/node": "22.14.0",
-    "@types/react": "^19.1.0",
-    "@types/react-dom": "^19.1.1",
+    "cross-env": "7.0.3",
```

Comment:
Why is this pinned to an exact version, and present here as well as in the site's dependencies?

Reply:
It's present b/c we use it in the `scripts` section of the `package.json`. In pnpm (IIUC), you need to declare dependencies in each `package.json` that they are used.

Reply:
Ah okay, that's news to me if that's the case -- can we double-check it is? If it is, can we have it reference the parent package's version for it?

And, I'm not sure that was an answer to why it is pinned to a specific version?